examples/Console/TestMetrics.cs

```diff
@@ -77,7 +77,7 @@ internal static object Run(MetricsOptions options)
                 providerBuilder
                     .AddConsoleExporter(o =>
                     {
-                        o.MetricExportIntervalMilliseconds = options.DefaultCollectionPeriodMilliseconds;
+                        o.PeriodicExportingMetricReaderOptions.ExportIntervalMilliseconds = options.DefaultCollectionPeriodMilliseconds;
```

Comment:
we need to set ReaderType to be periodic for this to take effect?

Reply:
Got it, set `MetricReaderType` to `periodic`.